
We later might like to disable more.

  - Rene Rebe <rene@exactcode.de>

--- embutils-0.17-patched/Makefile.orig	2005-02-02 13:12:10.851559944 +0100
+++ embutils-0.17-patched/Makefile	2005-02-02 13:12:15.720819704 +0100
@@ -10,7 +10,7 @@
 dirname domainname id ln mv cp yes which cat rm wc ls whoami mkfifo head \
 install sosrm soscp sosmv sosln soslns md5sum sleep2 allinone kill uniq \
 dd tr mesg write touch du tail uuencode uudecode nohup nice cmp mktemp \
-truncate strings test date mount printenv umount pivot_root insmod rmmod \
+truncate test date mount printenv umount pivot_root insmod rmmod \
 lsmod
 
 OBJDIR:=bin
